src/app/cdk/panel/panel-source/panel-source.component.html

```diff
@@ -24,10 +24,7 @@
         {{ sourceName }}
       </div>
     </div>
-    <div
-      class="preferred-source"
-      *ngIf="isPreferred && stackLength > 1 && !isPublicRecord"
-    >
+    <div class="preferred-source" *ngIf="isPreferred && stackLength > 1">
```

Comment:
@DanielPalafox This is reverting part of your work on https://github.com/ORCID/orcid-angular/commit/272794c1a38651f3460a75f4991c8c95a22637e7 please make sure I am not breaking your previous fix